notebooks/Notebook1.md

```diff
@@ -2,33 +2,51 @@
 
 ## Last week's critique
 
-**TODO:** Fill in this part with a summary and reflection on the critique you received for
-last week's work. Answer questions such as:  How, specifically, did the feedback help
-improve the project? Did the feedback point out or offer something you hadn't considered?
-Did it help you make a design decision? Was it helpful in addressing the most pressing
-issues in your project? How will you incorporate the feedback into your work? Will you
-change something about the design, implementation, or evaluation as a result?
+The critique had a lot of nice comments as well as some useful feedback! The analysis of my intended audience (writers with little to no CS experience) and having a full week for revisions and testing were highlighted as standouts of my pitch. Additionally, my thoughts about debugging/error handling and designing with possible expansion in mind were critiqued. I'd like to discuss what I've done with these last two this week below.
+
+Prioritizing custom error handling, both in parsing the DSL and when running the game is a great point. I've seen that the parsing package I'm using (pyparsing) has some decent error-checking built in, but I need to implement error handling for any processing that's done outside of that package - and right now, there's a decent amount of that. My goal is to learn more about the pyparsing package as the weeks go on and start to do more and more within the package so I'm better covered with errors there. 
+
+I also agree that while it's interesting to design my code base in mind for potential expansion, too much focus on that would distract from the core focus of DSL design and parsing. As the weeks go on, I'll focus my efforts in this space by making my code intuitive and readable, and not spend time making any specific structures to aid language expansion.
+
+
 
 ## Description
 
-**TODO:** Fill in this part with information about your work this week:
-important design decisions, changes to previous decisions, open questions,
-exciting milestones, preliminary results, etc. Feel free to include images
-(e.g., a sketch of the design or a screenshot of a running program), links to
-code, and any other resources that you think will help clearly convey your
-design process.
+I started by focusing in on room navigation. What I mean by this is when you are in the game, being able to say 'go south' or 'go west' and be able to navigate to that adjacent room when there is one present. I then sketched out the DSL design for room declaration, text, and navigation, where I settled on an HTML-like structure with directions specified in clear English. See the picture of my design below:
```

Comment:
I like how your language is consistent in that it takes the format "commandType object"; for example, go south. You mentioned also having commands like "look at object" which keeps this trend (though I might change the wording to "check object" for further consistency).